wiki/Help_Centre/Account_Restrictions/id.md

```diff
@@ -1,100 +1,115 @@
+- **Tunggu 1x24 jam.** Dalam kasus-kasus tertentu, sistem kami akan secara otomatis me-*restrict* pengguna-pengguna yang aktivitasnya dianggap mencurigakan. Berhubung seluruh *restriction* yang ada akan diverifikasi kembali oleh tim kami kemudian, harap tunggu 1x24 jam — apabila *restriction* yang kamu peroleh memang benar adanya diakibatkan oleh kesalahan sistem, tim kami akan mencabut status *restriction* akunmu dengan segera.
+- **Hubungi [account support team](/wiki/People/The_Team/Account_support_team) kami.** Apabila akunmu masih tetap di-*restrict* setelah 1x24 jam, harap kirimi kami email pada alamat [accounts@ppy.sh](mailto:accounts@ppy.sh) **dengan menggunakan alamat email yang kamu gunakan untuk membuat akunmu**. Pastikan kamu menyertakan nama penggunamu dan rincian seputar hal-hal apa saja yang kamu curigai menyebabkan akunmu di-*restrict* dalam email yang kamu kirim. Semakin lengkap informasi yang kamu berikan, semakin mudah bagi kami untuk dapat menyelesaikan masalahmu.
+- Apabila menurutmu *restriction* yang kamu peroleh merupakan kekeliruan, harap hubungi kami secepatnya agar kasusmu dapat dibicarakan lebih lanjut.
 
-Jika akun Anda di-restrict karena telah melanggar [peraturan osu!](/wiki/Rules), akan diterapkan masa tunggu selama tiga bulan sebelum Anda diizinkan mengajukan banding.
+Apabila akunmu di-*restrict* karena kamu telah melanggar [peraturan yang berlaku](/wiki/Rules), kamu akan diberikan masa tunggu selama tiga bulan sebelum kamu dapat mengajukan banding.
 
-Jika akun asli Anda tidak di-restrict, Anda dapat masuk dan terus bermain seperti biasa. Tetapi pastikan untuk tidak masuk menggunakan akun lain yang Anda miliki, karena dapat berujung ke sanksi yang lebih berat.
+Apabila akunmu di-*restrict* karena kamu memiliki lebih dari satu akun namun akun utama yang kamu miliki tidak di-*restrict*, kamu dapat tetap bermain di akun utama tersebut seperti biasa. Meskipun demikian, harap pastikan ke depannya kamu tidak kembali membuat atau menggunakan akun-akun lainnya karena tindakan ini dapat berujung pada sanksi-sanksi yang lebih berat.
 
-Jika pengajuan banding Anda baru saja dikabulkan namun akun Anda kembali di-restrict setelah bermain, jangan panik! Sistem mungkin telah salah menandai akun Anda secara otomatis, tetapi akun Anda tidak akan di-restrict lagi dalam kurun waktu 24 jam jika hal ini terbukti benar. Jangan ragu untuk mengirimkan kami email atau *tweet* di Twitter untuk menarik perhatian kami agar restriction akun dapat dicabut secepatnya!
+Apabila bandingmu dikabulkan namun akunmu kembali di-*restrict* sesaat setelah kamu kembali bermain, jangan panik! Hal ini kemungkinan disebabkan oleh sistem kami yang menandai akunmu sebagai akun yang mencurigakan secara tidak sengaja. Jangan ragu untuk mengontak kami via email atau Twitter agar kami dapat kembali memulihkan akunmu sesegera mungkin!
```

Comment:
Twitter account URL should be added in all language

Reply:
yup, i agree but i feel like that it should be done in a different pr instead after this is merged (otherwise it would be out of this pr's scope)

Reply:
correct